packages/wonder-blocks-button/package.json

```diff
@@ -7,6 +7,7 @@
   },
   "description": "",
   "main": "dist/index.js",
+  "module": "dist/es/index.js",
```

Comment:
Using the `module` field for `ES6` modules is one of the many ways of doing this (there's also `jsnext`).  rollup supports consuming modules with either.  I decided to go with `module` b/c it sounded nicer.

Reply:
Seems good to me!